tempesta_fw/t/functional/flacky_net/test_sockets.py

```diff
@@ -0,0 +1,147 @@
+"""All sockets must be closed after Tempesta shutdown"""
```

Comment:
Style guide in Python is not so flexible like in C. Python community forces every developer to use the  official code style described in PEP-8 document https://www.python.org/dev/peps/pep-0008 . Indentation in Python matters a lot since it is used instead old good operator brackets, so the community decided to have the only right variant: 4 spaces, tabs and mixing tabs with spaces are not allowed. Even more the wrong indentation is equal to syntax error in Python 3. `pylint`, the Python linter tool, threats any indentation different from 4 spaces as warnings.

I agree with you, that the whole project must use the same code style. But we can literately do nothing here.